### ISSUE TITLE
Why does npm postinstall run bower install?

### ISSUE THREAD

**Issue by dcosson** [c-trimm/moment-recur#47]
It doesn't work with the way I'm trying to install this (in a docker container - bower can't be run as root and the docker container only has root). Why does it need bower?

**Comment by ZombiEquinox** [comment]
I am trying to install VIA NPM onto a node server, and I am getting the same issue. moment-recur will not install because of the preinstall. I've tried installing just the two dependencies by themselves, and it still won't install.

**Comment by bmathews** [comment]
Can be circumvented by adding a `.bowerrc` file to your project root with `{ "allow_root": true }`

**Comment by tphummel** [comment]
`gulp` is already in `devDependencies`. why are you installing it again as a preinstall script? can `bower` be added to `devDependencies`, too? or is there some reason you put it in `preinstall`? i'm referring to this:

```
  "scripts": {
    "test": "gulp test",
    "preinstall": "npm install bower gulp",
    "postinstall": "bower install"
  },
```

**Comment by jrgd** [comment]
thanks @bmathews that did the trick

**Comment by paglias** [comment]
This makes it impossible to run `npm install && npm shrinkwrap` because NPM will see gulp and bower as extraneous deps and error

**Comment by kilianc** [comment]
This is still a bug even with the workaround. the preinstall and postinstall should be ran before publishing to npm and we should only get the module's files when we require. I don't have bower on my container and this makes it impossible to use the library :(

**Comment by ccuilla** [comment]
You may want to move away from bower anyway:

```
npm WARN deprecated bower@1.8.0: ..psst! While Bower is maintained, we recommend Yarn and Webpack for *new* front-end projects! Yarn's advantage is security and reliability, and Webpack's is support for both CommonJS and AMD projects. Currently there's no migration path but we hope you'll help us figure out one.
```

**Comment by fubar** [comment]
This is exactly why I created this: https://www.npmjs.com/package/moment-recur-npm / https://github.com/fubar/moment-recur-npm

(I still need to pull the recent changes into it.)

**Comment by c-trimm** [comment]
This is now fixed in 1.0.7 - pre and post install scripts have been removed.
